client/src/styles/App.css

```diff
@@ -5,11 +5,14 @@
 }
 
 :root {
-  --primary-color: #695e6e;
-  --secondary-color: #f4f4f4;
-  --third-color: #ffd9bd;
-  --fourth-color: #252323;
+  --primary-font-color: #252323;
+  --secondary-font-color: #f4f4f4;
+  --primary-color: #ffd44c;
+  --secondary-color: #00034d;
+  --third-color: #f165d2;
+  --fourth-color: #f4f4f4;
+  --fifth-color: #252323;
   --font-title: "Sedgwick Ave", sans-serif;
-  --font-subtitle: "Sedan", sans-serif;
-  --font-content: "Inter", sans-serif;
+  --font-subtitle: "Atkinson Hyperlegible", sans-serif;
```

Comment:
Why is the same font displayed 2 times ?

Reply:
Je ne comprends pas pourquoi, quand je regarde mon fichier j'ai à la ligne 14 et 15 ce qui suit:
  --primary-font: "Sedgwick Ave display", sans-serif;
  --secondary-font: "Atkinson Hyperlegible", sans-serif;
rien d'autres